src/biotite/structure/info/atoms.py

```diff
@@ -30,6 +30,11 @@ def residue(res_name):
     ----------
     res_name : str
         The up to 3-letter name of the residue.
+    allow_missing_coords: bool
+        Whether to allow missing coordinate values in the residue.
+        If `True`, these will be represented as `nan` values.
+        If `False`, a `ValueError` is raised when missing coordinates
+        are encountered.
```

Comment:
In contrast to Markdown Sphinx requires double quoted values to render them as Markdown
```suggestion
        If ``True``, these will be represented as ``nan`` values.
        If ``False``, a ``ValueError`` is raised when missing coordinates
        are encountered.
```